sklearn/metrics/tests/test_classification.py

```diff
@@ -178,9 +179,9 @@ def test_classification_report_output_dict_empty_input():
             "support": 0,
         },
         "weighted avg": {
-            "f1-score": 0.0,
-            "precision": 0.0,
-            "recall": 0.0,
+            "f1-score": np.nan,
```

Comment:
This looks like a backward incompatible change for `classification_report`.

Reply:
@thomasjpfan well, from my perspective it's a bug fix. Why `macro avg` would return f1-score=np.nan and `weighted avg` 0.0, when there's no input. Should not they return the same?

Reply:
Yea, that makes sense. I am okay with treating this as a bug fix. Can we add another entry on the change log so users know about the behavior change in `classification_report`?

Reply:
done